### DISCUSSION TITLE
Looking for team members

### DISCUSSION THREAD

**Discussion by Sudkorea** [irgroup-classrooms/dis17-2021#4] [General]
Introduce me

Bad things
An exchange student who is terrible at German-speaking tried to use English, and expert at Korean.

Good things
I can hear English things.
Major in mathematics at the University of Seoul.
Had a GODLIKE grade(A or A+) in ODE, Abstract Algebra, Topology, and Analysis.
Had a good grade(B or B+) in Linear algebra, PDE, Discrete mathematics, Numerical analysis, and Set theory.
Love math.

Whatsapp : +4917680400931

**Comment by breuert** [comment]
Hi @Sudkorea - have you found a team yet?

**Reply by Sudkorea** [reply]
No, I don’t have a team yet..

**Reply by breuert** [reply]
Thanks for the update! We'll find one team for you tomorrow! :)
